modules/ROOT/pages/raw-data.adoc

```diff
@@ -1,7 +1,7 @@
 = Downloading Raw Data
 :page-info-pills: anypoint-pricing-titanium
 
-You can use Anypoint Monitoring to download archived log data (raw data) collected on Mule apps that are running in hybrid, CloudHub, or Runtime Fabric deployments. This data is helpful for security, compliance, or other purposes where you need to examine or collect your warehoused data.
```

Comment:
No need to review: Just clarifying this applies only to US & EU. No plans to do improvements to this page at this time.